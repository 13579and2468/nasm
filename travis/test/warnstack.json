[
	{
		"description": "Test warning stack",
		"id": "warnstack",
		"source": "warnstack.asm",
		"option": "-E",
		"target": [
			{ "stdout": "warnstack.stdout" },
			{ "stderr": "warnstack.stderr" }
		]
	}
]
