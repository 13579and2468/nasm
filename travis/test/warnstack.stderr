./travis/test/warnstack.asm:1: warning: Good warning [-w+user]
./travis/test/warnstack.asm:4: warning: Bad warning [-w+user]
./travis/test/warnstack.asm:6: warning: Good warning [-w+user]
./travis/test/warnstack.asm:8: warning: Bad warning [-w+user]
./travis/test/warnstack.asm:10: warning: Good warning [-w+user]